### ISSUE TITLE
[Bug] Pydeck error in Google Colab 

### ISSUE THREAD

**Issue by Jesus89** [visgl/deck.gl#7280]
### Description

When rendering a pydeck map in Google Colab, the map is rendered, but then it raises the following error:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[/usr/local/lib/python3.7/dist-packages/IPython/core/formatters.py](https://localhost:8080/#) in __call__(self, obj)
    343             method = get_real_method(obj, self.print_method)
    344             if method is not None:
--> 345                 return method()
    346             return None
    347         else:

[/usr/local/lib/python3.7/dist-packages/pydeck/bindings/deck.py](https://localhost:8080/#) in _repr_html_(self)
    221         # doesn't actually need the HTML packaging in iframe_with_srcdoc,
    222         # so we just take the HTML.data part
--> 223         return self.to_html(notebook_display=True).data

AttributeError: 'str' object has no attribute 'data'
```

This only happens when rendering the Deck object, not with the `to_html()` function, but in this case, it returns the HTML code, which I think it's not convenient.

I think this bug comes in two steps:
- Return html_str in colab: https://github.com/visgl/deck.gl/pull/5657/files#diff-4da9d85cf3adb9c98acd2c74a17bf1563587afe99e1ea93e2155c054901fe9eeR156
- Use to_html().data as repr_str: https://github.com/visgl/deck.gl/pull/5486/files#diff-bfa722fd9c40db85f3622f615422be4e4234fc86f427251ffa429612f2a44d03R195

IMO, the fix is just to remove [this line](https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/pydeck/io/html.py#L156), but would like to contrast with @ajduberstein. 

### Flavors

- [ ] React
- [X] Python/Jupyter notebook
- [ ] MapboxLayer
- [ ] GoogleMapsOverlay
- [ ] CartoLayer
- [ ] DeckLayer/DeckRenderer for ArcGIS

### Expected Behavior

Render pydeck maps in *Google Colab* without the error message, or the HTML output.

### Steps to Reproduce

Execute this code in a *Google Colab* notebook:

```py
import pydeck as pdk

UK_ACCIDENTS_DATA = 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv'

layer = pdk.Layer(
    'HexagonLayer',  # `type` positional argument is here
    UK_ACCIDENTS_DATA,
    get_position=['lng', 'lat'],
    auto_highlight=True,
    elevation_scale=50,
    pickable=True,
    elevation_range=[0, 3000],
    extruded=True,
    coverage=1)

# Set the viewport location
view_state = pdk.ViewState(
    longitude=-1.415,
    latitude=52.2323,
    zoom=6,
    min_zoom=5,
    max_zoom=15,
    pitch=40.5,
    bearing=-27.36)

# Combined all of it and render a viewport
pdk.Deck(layers=[layer], initial_view_state=view_state)
```

### Environment

- Framework version: deck.gl@8.8.11
- Browser: Chrome 104.0
- OS: Ubuntu 20.04.4 LTS

**Comment by kylebarron** [comment]
> This only happens when rendering the `Deck` object, not with the `to_html()` function, but in this case, it returns the HTML code, which I think it's not convenient.
> 
> ```py
> pdk.Deck(layers=[layer], initial_view_state=view_state).to_html()
> ```

Am I misreading this? You said the issue happens when rendering the `Deck` object, but your example uses `.to_html()`?

Is pydeck erroring when used as a widget or an html blob?

**Comment by Jesus89** [comment]
You're right, the example shouldn't contain `to_html()`, I pasted the working version. Updated now with the non-working version.

**Comment by kylebarron** [comment]
Seems like the right approach is to stop defaulting to html rendering in colab and opting in to their ipywidgets support https://github.com/googlecolab/colabtools/issues/498#issuecomment-998308485

**Comment by ajduberstein** [comment]
In the near-term @kylebarron's suggestion seems best -- we should opt into Colab's widgets support. We've had bidirectional support for Colab in [the release tracker](https://github.com/visgl/deck.gl/issues/6142) for the last few releases, would love to see this added.

That said, if the change is more difficult than it seems, I'm fine with the fix @Jesus89 proposes here --

> IMO, the fix is just to remove [this line](https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/pydeck/io/html.py#L156)

@Jesus89, feel free to modify and I'll stamp the PR.

**Comment by Jesus89** [comment]
Hi @ajduberstein

I tested with Google Colab widgets and found the following error:
![](https://user-images.githubusercontent.com/2227572/195514476-0da5ed0a-4989-4516-9d3b-3f9c7168f65d.png)

I didn't spend more time on that, so I focused on fixing the current bug and increasing test coverage. I agree that in the long term pydeck should have better integration with Colab widgets, but for now, the bug is fixed to move forward with a stable release. Thanks in advance for the review.
